charts/tfy-gpu-operator/Chart.yaml

```
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: tfy-gpu-operator
-version: 0.1.12
+version: 0.1.13
```

If we are not yet finalized with values and experimenting then please keep it rc

```suggestion
version: 0.1.13-rc.1
```